rllib/evaluation/rollout_worker.py

```diff
@@ -1727,7 +1730,7 @@ def _make_sub_env_local(vector_index):
             # Create the sub-env.
             env = env_creator(env_ctx)
             # Validate first.
-            _validate_env(env, env_context=env_ctx)
+            check_env(env)
```

Comment:
Quick question: Why don't we have to do:
```
if not disable_env_checking:
                 check_env(env)
```
here, like we do above?

Reply:
when does this function get called? I can't find an example that calls this function